### PR TITLE
Update README with EMP details

### PR DESCRIPTION
**Pull request by muneebahmed10**
*Description*:
Update the README with details on the FreeRTOS Extendend Maintenance Plan.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
...
+FreeRTOS offers feature stability with long term support (LTS) releases. FreeRTOS LTS libraries come with security updates and critical bug fixes to the FreeRTOS kernel and IoT libraries listed below for two years, and are maintained by AWS for the benefit of the FreeRTOS community. With FreeRTOS LTS, you get a complete set of libraries needed to build secure connected IoT and embedded products. Long term support helps reduce maintenance and testing costs associated with updating libraries on your devices already in production.
+
+AWS also offers FreeRTOS Extended Maintenance Plan (EMP) that provides you with security patches and critical bug fixes on your chosen FreeRTOS LTS version for up to an additional 10 years. With FreeRTOS EMP, your FreeRTOS-based long-lived devices can rely on a version that has feature stability and receives security updates for years. You receive timely notification of upcoming patches on FreeRTOS libraries, so you can plan the deployment of security patches on your Internet of Things (IoT) devices. To learn more about FreeRTOS EMP, see the [Features page](https://aws.amazon.com/freertos/features/).
```
Comment (tanmoysen) [inline README.md:4]: We can just use IoT devices in the last sentence (not Internet of Things (IoT) devices). Otherwise, looks good. Thank you!

Comment (tanmoysen) [inline README.md:4]: also, "see the FreeRTOS Features page"-  last sentence.

Comment (muneebahmed10) [reply README.md]: Changed

Comment (muneebahmed10) [reply README.md]: Done

**Review by tanmoysen — APPROVED** [review]

**Review by paulbartell — APPROVED** [review]

**Pull request merged**